src/bust.py

```diff
@@ -387,10 +417,27 @@ async def create_controller(
             list_message = await message.channel.send(embed=embed)
             message_list.append(list_message)
 
-        # If message channel == target channel, pin messages in reverse order
-        if target_channel == message.channel:
+        # If message channel == target channel, or we ask for a mock,
+        # pin messages in reverse order and generate Google Form
+        pin_and_form = (
+            target_channel == message.channel or "full" in message.content.split()
+        )
+        if pin_and_form:
             for list_message in reversed(message_list):
                 await discord_utils.try_set_pin(list_message, True)
+            # Wrap form generation in try/catch so we don't block !list if it fails
+            form_url = None
+            try:
+                form_url = await bc.get_google_form_url(image_url)
+            except Exception as e:
+                print("Unknown error generating form", e)
+
+            if form_url is not None:
+                vote_emoji = ":ballot_box:"
```

Comment:
To be consistent with the definitions in `config.py`, this can also be `"\N{BALLOT BOX WITH BALLOT}"`.